### PR TITLE
Update docs for the CMS HCC mart

### PR DESCRIPTION
**Pull request by sarah-tuva**
* Updated the docs page for CMS HCC mart
* Added SQL examples for CMS HCC

### CODE REVIEW

File: docs/data-marts/cms-hcc/example-sql.md
Comment (aneiderhiser) [inline docs/data-marts/cms-hcc/example-sql.md:1]: We want this to show up in the data mart drop-down for CMS-HCCs, right?  It's not currently showing up.  Need to add it in sidebars.js

**Review by aneiderhiser — CHANGES_REQUESTED** [review]
Looks good except for example sql which i left a comment on.

File: docs/data-marts/cms-hcc/example-sql.md
Comment (sarah-tuva) [reply docs/data-marts/cms-hcc/example-sql.md:1]: I'm confused. It's showing up in both places. There were already placeholders for it so I just updated the page. Are you not seeing it in the sidebar under Data Marts? 
image

Comment (sarah-tuva) [reply docs/data-marts/cms-hcc/example-sql.md:1]: Oops, nevermind. I see the problem! 😅

**Review by aneiderhiser — APPROVED** [review]
Looks great to me!

**Pull request merged**